README.md

```diff
@@ -438,3 +438,18 @@ npm run dev -- <command> <options>
 ```
 npm run test
 ```
+
```

Comment:
I thought you wanted to run in the CI, but I don't see anything ?

Reply:
In the end, I find that it is weird to place it in CI since the npm run test has covered the wrap functionality testing. I believe CI is used to perform a check for fault isolation and identifying problematic patterns. The performance testing script doesn't do any of the categories, it just shows the wrapping response time that is kind of pointless.

Reply:
yes and no. If it breaks for whatever reason we will never know. Let's run  the wrap with 2 files and one iteration at least we know it works

Reply:
done